WORKSPACE

```diff
@@ -437,11 +443,12 @@ http_archive(
     name = "httplib2",
     build_file_content = """
 py_library(
-name = "httplib2",
-srcs = glob(["**/*.py"]),
-data = ["cacerts.txt"],
-visibility = ["//visibility:public"]
-)""",
+    name = "httplib2",
```

Comment:
this change is nice, but not strictly needed to fix this issue right? I'm having trouble understanding what problem you discovered from your investigation

Reply:
It's not related to the fix. There were two commits on this PR, one contained the fix and the other was cleanup. I've since dropped the second commit.